addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -53,6 +53,10 @@ def _default_default_message(self):
         string="Maximum Sessions",
         help="Maximum number of concurrent sessions per operator.",
     )
+    buffer_time = fields.Integer(
+        default=0,
+        help="Time in seconds between two consecutive sessions. This is used to avoid that an operator is assigned to a new session while he is still in the preliminary investigation of the former one.",
```

Comment:
:pick: 

Shorter string, and maybe split on several lines if still too long? Also from now on "agent" is the preferred wording over "operator".

`Time in seconds between two sessions, to prevent assigning a new one to an agent who may still be handling the previous one.`